services/brig/src/Brig/API.hs

```diff
@@ -72,17 +72,31 @@ sitemap o = do
 
   get "/i/status" (continue $ const $ return empty) true
   head "/i/status" (continue $ const $ return empty) true
+  -- This endpoint can lead to the following events being sent to clients:
+  -- - ConnectionUpdated event to the user and all users connecting with
+  -- - ConvCreate event to the user for each connect conversation that did not exist before
+  --   (via galley)
+  -- - ConvConnect event to the user for each connection that was not already accepted by the
+  --   other
+  -- - MemberJoin event to the user and other for each connection that was not already
+  --   accepted by the other
   post "/i/users/:uid/auto-connect" (continue autoConnectH) $
     accept "application" "json"
       .&. capture "uid"
       .&. opt (header "Z-Connection")
       .&. jsonRequest @UserSet
+  -- This endpoint can lead to the following events being sent to clients:
+  -- - UserActivated event to created user, if it is a team invitation or user has an SSO ID
+  -- - UserIdentityUpdated event to created user, if email or phone get activated
   post "/i/users" (continue createUserNoVerifyH) $
     accept "application" "json"
       .&. jsonRequest @NewUser
   put "/i/self/email" (continue changeSelfEmailNoSendH) $
     header "Z-User"
       .&. jsonRequest @EmailUpdate
+  -- This endpoint will asynchronously lead to the following events being sent to clients:
```

Comment:
What does asynchronously mean here? What if that fails half way through?

Reply:
An "internal event" is put into a queue, usually SQS (see `Brig.Queue`). 

I could either clarify that by pointing to `Brig.Queue` or just remove the information about his happening asynchronously, as it's not directly relevant to the events being sent.

What do you prefer?

Reply:
I'll leave out the note about it being asynchronous.

Reply:
done